BotProject/Templates/CSharp/Program.cs

```diff
@@ -26,8 +26,9 @@ public static void Main(string[] args)
             {
                 var env = hostingContext.HostingEnvironment;
 
-                builder.AddJsonFile($"ComposerDialogs/settings/appsettings.json", optional: true, reloadOnChange: true)
-                    .AddJsonFile("appsetting.json", optional: true, reloadOnChange: true)
+                var configFile = Path.GetFullPath(Path.Combine(@"../settings/appsettings.json"));
```

Comment:
We should not do this, we need to get the bot path first, and then search in botfolder/settings/appsettings.json, otherwise it will not working for the deployment scenario.

Reply:
And, we should consider the deployment scenario, if it is deployment, we need to get the bot and root location from appsettings.deployment.json.

Reply:
> We should not do this, we need to get the bot path first, and then search in botfolder/settings/appsettings.json, otherwise it will not working for the deployment scenario.

fixed

Reply:
> And, we should consider the deployment scenario, if it is deployment, we need to get the bot and root location from appsettings.deployment.json.

currently the deployment bot's path is hard-coded, do we need to make that configurable, I mean, move the 'bot': '../' to appsettings.development.json layer, and add 'bot': 'composerDialogs' to appsettings.deployment.json layer.

Reply:
yes, we should make them configurable